declarative/declarativemprisplayer.cpp

```diff
@@ -22,6 +22,420 @@
+
+    Should be set to true if the current playlist is being played in random
+    order.
+
+    Defaults to false.
+*/
+
+/*!
+    \qmlproperty double MprisPlayer::volume
+    \brief Indicates the current playing volume of the player
+
+    Should be set to current relative playing volume used by the player.
+
+    Defaults to 1.0.
+*/
+
+/*!
+    \qmlsignal MprisPlayer::quitRequested()
+
+    This signal is emitted when there is an incoming request
+    to close the application.
+*/
+
+/*!
+    \qmlsignal MprisPlayer::raiseRequested()
+
+    This signal is emitted when there is an incoming request
+    to present the application to the user.
+*/
+
+/*!
+    \qmlsignal MprisPlayer::loopStatusRequested(int loopStatus)
+
+    This signal is emitted when there is an incoming request
+    to change the way the current media playlist is to be repeated.
+
+    \sa MprisPlayer::loopStatus
+*/
+
+/*!
+    \qmlsignal MprisPlayer::rateRequested(double rate)
+
+    This signal is emitted when there is an incoming request to change
+    the playback rate.
+
+    \sa MprisPlayer::rate
+    \sa MprisPlayer::minimumRate
+    \sa MprisPlayer::maximumRate
+*/
+
+/*!
+    \qmlsignal MprisPlayer::shuffleRequested(bool shuffle)
+
+    This signal is emitted when there is an incoming request to change
+    whether the current media playlist should be played in random order.
+
+    \sa MprisPlayer::shuffle
+*/
+
+/*!
+    \qmlsignal MprisPlayer::volumeRequested(double volume)
+
+    This signal is emitted when there is an incoming request to change
+    the current playback volume.
+
+    \sa MprisPlayer::volume
+*/
+
+/*!
+    \qmlsignal MprisPlayer::nextRequested()
+
+    This signal is emitted when there is an incoming request to move
+    to the next item on the playlist.
+
+    \sa MprisPlayer::canGoNext
+*/
+
+/*!
+    \qmlsignal MprisPlayer::openUriRequested(url uri)
+
+    This signal is emitted when there is an incoming request to
+    play media from specified location.
+
+    \sa MprisPlayer::supportedUriSchemes
+    \sa MprisPlayer::supportedMimeTypes
+*/
+
+/*!
+    \qmlsignal MprisPlayer::pauseRequested()
+
+    This signal is emitted when there is an incoming request to
+    pause current playback.
+
+    \sa MprisPlayer::canPause
+*/
+
+/*!
+    \qmlsignal MprisPlayer::playRequested()
+
+    This signal is emitted when there is an incoming request to
+    start or resume playback.
+
+    \sa MprisPlayer::canPlay
+*/
+
+/*!
+    \qmlsignal MprisPlayer::playPauseRequested()
+
+    This signal is emitted when there is an incoming request to
+    start, resume or pause current playback.
+
+    \sa MprisPlayer::canPlay
+    \sa MprisPlayer::canPause
+*/
+
+/*!
+    \qmlsignal MprisPlayer::previousRequested()
+
+    This signal is emitted when there is an incoming request to move
+    to the previous item on the playlist.
+
+    \sa MprisPlayer::canGoPrevious
+*/
+
+/*!
+    \qmlsignal MprisPlayer::seekRequested(int offset)
```

Comment:
This and the following ones qlonglong in signals, int here losing precision? Also affected by the qml type conversions.

Reply:
Changed.